Apps/Database/src/Models/Dependent.cs

```diff
@@ -0,0 +1,39 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.Database.Models
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using System.ComponentModel.DataAnnotations.Schema;
+
+    /// <summary>
+    /// The dependent model.
+    /// </summary>
+    public class Dependent : AuditableEntity
```

Comment:
Instead of dependent, how about it being a Delegate table. That why the relation is abstracted

Reply:
I know you're aiming for the future features like UMA, but i'm pretty sure features like that would include more changes to the whole functionality, we can discuss the name during the grooming call this afternoon

Reply:
yeh, sounds good 👍

Reply:
Changing the name of the table later would be pretty painful, I think calling it Delegate will make our lives easier in the long run.  Based on that, I also think the field names would be DelegateHdId (or HdId as Delegate is redundant) and UserProfileHdId to identify the relationship exactly.